staging/src/k8s.io/apimachinery/pkg/util/intstr/intstr.go

```diff
@@ -169,6 +169,8 @@ func GetValueFromIntOrPercent(intOrPercent *IntOrString, total int, roundUp bool
 	return value, nil
 }
 
+//Deprecated: use getIntOrPercentValueFromIntOrString
+//lint:ignore U1000 we want to keep the changes from breaking external uses of the method
```

Comment:
`GetValueFromIntOrPercent` should still call this to avoid breaking external users, this lint exception shouldn't be required